### ISSUE TITLE
Release ytt v0.37.0

### ISSUE THREAD

**Issue by pivotaljohn** [k14s/ytt#502]
### Release Checklist
- [x] Create release in Github
    - [x] [Bump Version](https://hackmd.io/uVpvITUuR4Cbwzkzb7MEpQ#Bump-version)
    - [x] [Generate Binaries and Checksum](https://hackmd.io/uVpvITUuR4Cbwzkzb7MEpQ#Generate-Binaries-and-Checksum)
    - [x] [Create a Github release](https://hackmd.io/uVpvITUuR4Cbwzkzb7MEpQ#Create-a-Github-Release)
    - [x] [Create the Release Notes](https://hackmd.io/uVpvITUuR4Cbwzkzb7MEpQ#Create-the-Release-Notes)
    - [x] [Publish the Draft Release](https://hackmd.io/uVpvITUuR4Cbwzkzb7MEpQ?view#Publish-the-draft-release)
    - [x] [Create an OSM Release](https://hackmd.io/uVpvITUuR4Cbwzkzb7MEpQ#OSM-Release)
- [x] Ask Dmitriy to deploy this release to the Playground.
- [x] Close any GitHub issues that have been delivered.
  - [x] Add a link to the release on the issue.
- [x] Update the packaging repositories
    - [x] [Update Homebrew](https://hackmd.io/uVpvITUuR4Cbwzkzb7MEpQ#Update-Homebrew)
    - [x] [Update Github Action](https://hackmd.io/uVpvITUuR4Cbwzkzb7MEpQ#Update-Github-Action)
    - [x] [Update Website Installation Script](https://hackmd.io/uVpvITUuR4Cbwzkzb7MEpQ#Update-Website-Installation-Script)
- [x] [Create TanzuNet product](https://hackmd.io/uVpvITUuR4Cbwzkzb7MEpQ#Create-TanzuNet-product)
- [x] [Add announcement in community meeting](https://hackmd.io/uVpvITUuR4Cbwzkzb7MEpQ#Announce-in-community-meeting)
- [x] update `ytt` docs
- [x] [Let Nanci know we're ready to announce](https://hackmd.io/uVpvITUuR4Cbwzkzb7MEpQ#Communicate-in-Slack)
- [x] Confirm Playground has the new version.

Post Release:
- [x] Notify Chocolatey maintainer of new release
- [x] (~ 2021-10-21) Upload and release v0.37.0 OSL file to TanzuNet Product and Release

**Comment by pivotaljohn** [comment]
At one point we ran into a problem where our Norsk pipeline failed.

TL;DR. `ytt`'s pipeline got moved from the `norsk-viewers` team to the `shared-norsk-viewers`. But the pipeline move wasn't completed.

Details:
- failing pipeline: https://norsk.cf-app.com/teams/norsk-viewers/pipelines/ytt/jobs/vmware-tanzu-carvel-ytt-report/builds/61
- likely cause: https://github.com/pivotal-cf/norsk-config/commit/b2830b60d7e1353b1526dcad3e62238848bb4b39
- slack convo with folks to resolve: https://vmware.slack.com/archives/C0186SB6ATY/p1633719490454300

---
We updated the Release docs to reflect the pipeline we should watch from now on.

**Comment by pivotaljohn** [comment]
As of now, just waiting for `ytt` playground to be redeployed: https://vmware.slack.com/archives/C010XR15VHU/p1633715548415800

**Comment by cppforlife** [comment]
@pivotaljohn redeployed.

**Comment by pivotaljohn** [comment]
OSL file attached to TanzuNET product.
